observations/obs_converters/even_sphere/README.rst

```diff
@@ -2,75 +2,277 @@
+
+Examples
+--------
+
+1. 30 horizontal locations at 6 pressure levels:
+
+.. code-block::
+
+   nprofiles   = 30;
+   levels      = [1000  850  500  300  200  100];
+   T_error_var = [1.44 0.64 0.64 0.81 1.44 0.64];
+   W_error_var = [1.96 2.25 4.41 9.00 7.29 4.41];
+   even_sphere(nprofiles, 'levels', levels, ...
+              'T_error_var', T_error_var, 'W_error_var', W_error_var)
+
+
+2. 30 horizontal locations at 3 pressure levels. Note that the
+   *nlevels* argument specifies that only the first 3 pressure levels
+   are used even though there are 6 potential pressure levels. 
+   Similarly, only the matching error variances are used.
+
+.. code-block::
+
+   nprofiles   = 30;
+   nlevels     = 3 ;
+   levels      = [1000  850  500  300  200  100];
+   T_error_var = [1.44 0.64 0.64 0.81 1.44 0.64];
+   W_error_var = [1.96 2.25 4.41 9.00 7.29 4.41];
+   even_sphere(nprofiles, 'nlevels', nlevels, 'levels', levels, ...
+              'T_error_var', T_error_var, 'W_error_var', W_error_var)
+
+Levels
+------
+ 
+The *mandatory pressure levels* defined in the
+`AMS glossary <https://amsglossary.ametsoc.org/wiki/Mandatory_level>`_ and their corresponding error variances are
+
+.. code::
+
+   levels      = [1000  925  850  700  500  400  300   250  200  150  100   70   50   30   20   10    7    5    3    2    1];
+   T_error_var = [1.44 1.00 0.64 0.64 0.64 0.64 0.81  1.44 1.44 1.00 0.64 0.64 0.81 1.00 1.69 2.25 2.25 2.25 2.25 2.25 2.25];
+   W_error_var = [1.96 2.25 2.25 2.56 4.41 6.76 9.00 10.24 7.29 5.76 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41];
+
+The default levels that this program generates were use to create the 
+Zagar OSSE which did not assimilate any observations above (about) 36 hPa.
+The default pressure levels and corresponding observation error variances are: 
+
+.. code::
+
+   levels      = [1000  925  850  700  500  400  300   250  200  150  100   70   50   40];
+   T_error_var = [1.44 1.00 0.64 0.64 0.64 0.64 0.81  1.44 1.44 1.00 0.64 0.64 0.81 1.00];
+   W_error_var = [1.96 2.25 2.25 2.56 4.41 6.76 9.00 10.24 7.29 5.76 4.41 4.41 4.41 4.41];
+
+.. attention::
+
+   Specifying the vertical levels is the most problematic part of this process.
+   Since observation error variances tend to vary with level, coordinating
+   the levels and the error variances is required.
+
+
+Automation Scripts
+------------------
+
+Here there are also scripts (*run_fixed_network_\*.csh*) which use the
+output from create_obs_sequence and the 
+:doc:`../../../assimilation_code/programs/create_fixed_network_seq/create_fixed_network_seq` 
+to generate a series of observation sequence files.
+
+
+run_fixed_network_seq.csh
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+run_fixed_network_daily.csh
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The process, end to end:
 
 MATLAB:
 
-edit even_sphere.m and set the number of levels, the
+Edit even_sphere.m and set the number of levels, the
```

Comment:
i guess with tim's changes this instruction has to change?

Reply:
I am still in the process of editing the README.rst and figuring out how it plays with the other scripting.
I really just wanted to give people an indication of the direction I am taking with even_sphere.m and how I envision it being used.
I have not gotten to the scripting aspect of how it might be used.  Perhaps I should have left a 'FEEL FREE TO IGNORE ANYTHING BELOW HERE'  right after the Levels section of the README.rst.